src/panels/lovelace/editor/config-elements/hui-statistics-graph-card-editor.ts

```diff
@@ -41,11 +42,21 @@ const cardConfigStruct = assign(
     entities: array(entitiesConfigStruct),
     title: optional(string()),
     days_to_show: optional(number()),
+    period: optional(
+      union([
+        literal("5minute"),
```

Comment:
Shouldn't this be "5minute**s**" (plural)?

Reply:
Then we would differ from the backend options, and I don't think we should do that. The UI will show minutes.

Reply:
Ah, wasn't aware that the backend defines that value.